Configure

```diff
@@ -269,12 +269,12 @@ my @disablables = (
     "ec_nistp_64_gcc_128",
     "egd",
     "engine",
-    "err",			# Really???
+    "err",
     "heartbeats",
     "hmac",
     "hw(-.+)?",
     "idea",
-    "locking",			# Really???
+    "locking",
```

Comment:
That is literally a no-op, since it only removes the 'Really???' comment.

The `no-locking` configure option exists both before and after this parth, and OPENSSL_NO_LOCKING still exists in `include/openssl/crypto.h`.

Using `no-locking` seems to still save me about 19KiB on the UEFI build, although I'm a little surprised it's that high.